travel_platform/izi_travel_client.py

```diff
@@ -0,0 +1,412 @@
+        return detail_museum_info
+
+    def get_tourist_attractions(self, tour_uuid, params=None):
+        """
+        Get list of tourist attractions included in a tour.
+        Example response:
+        [
+        {"status": .., "category": .., "placement": "..",
+        "uuid": "...", "distance": "..", "country": {...},
+        "languages": [..], "reviews": {..}, "map": {...},
+        "duration": ..., "content": [
+            {"language": "..", "title": "...", "playback": "..."},
+            "images": [{..}, {..}], "children": [
+            {...}, {...}, {...}]
+            ]
+        ]
+        Args:
+            tour_uuid (str): Tour uuid.
+            params (dict): Any additional url parameters.
+        Returns:
+            list: Tourist attractions.
+        """
+        url = self._prepare_base_url(self.TOUR_ENDPOINT.format(
+            tour_uuid=tour_uuid))
+
+        if params is None:
+            prepared_params = {"includes": "all,city,country",
+                               "except": "translations,publisher,download"}
+            params = self._prepare_params(**prepared_params)
+        response = self._make_request(url, **params)
+        tourist_attractions = []
+
+        if response:
+
+            tour_data = response.pop().get("content", []).pop()
+            for child in tour_data.get("children", []):
+
+                if child.get("type") == "tourist_attraction":
+                    attraction = {
+                        "title": child.get("title", ""),
+                        "description": child.get("desc", ""),
+                        "location": child.get("location", {}),
+                        "language": child.get("language", ""),
+                        "images": child.get("images", []),
+                        "attr_uuid": child.get("uuid", "")
+                    }
+                    tourist_attractions.append(attraction)
+            return tourist_attractions
+        return tourist_attractions
+
+    def get_object_reviews_and_rating(self, obj_uuid):
+        """
+        General method for getting reviews on any object by uuid.
+        Args:
+            obj_uuid (str): Object uuid.
+        Returns:
+            dict: Reviews info.
+        """
+        url = self._prepare_base_url(self.REVIEW_ENDPOINT.format(
+            obj_uuid=obj_uuid))
+        params = self._prepare_params()
+        response = self._make_request(url, **params)
+
+        if response:
+            reviews = response.get("metadata", {})
+            return reviews
+        reviews = {}
+        return reviews
+
+    def get_featured_content(self):
+        """
+        Returns:
+            dict: Website featured content.
+        """
+        url = self._prepare_base_url(self.FEATURED_ENDPOINT)
+        params = self._prepare_params()
+        response = self._make_request(url, **params)
+        return response
+
+
+# API_KEY = config.get('api.izi.travel', 'API_KEY')
```

Comment:
Looks like these prints and tests are useless here after adding of unittests. 
@kate-ka , What do you think about that?